core/templates/domain/user/user-info.model.ts

```diff
@@ -34,16 +34,19 @@ export class UserInfo {
   _isTopicManager: boolean;
   _isSuperAdmin: boolean;
   _canCreateCollections: boolean;
-  _preferredSiteLanguageCode: string;
-  _username: string;
-  _email: string;
+  // The default value of the following three properties of 'UserInfo' object
```

Comment:
It suffices to say "The following three properties are set to null when the user is not logged in." That is sufficient justification which constructively explains the meaning of a "null" value here.

Reply:
Have updated the comment to be a little shorter.